gencerts/gencerts.go

```diff
@@ -199,21 +197,26 @@ func fail(format string, a ...interface{}) {
 
 func indentBytes(data []byte) string {
 	var out []byte
-	idt := strings.Repeat(" ", indent)
+	idt := strings.Repeat("\t", indent)
 
 	s := fmt.Sprintf("%#v", data)
+	fist := true
```

Comment:
s/fist/first/ ?
In any case this would probably be slightly cleaner as 

```golang
for line := 0; len(s) > indentWrap; line++ {
  if sp := strings.IndexByte(s[indentWrap:], ','); sp > -1 {
    if line > 0 {
      out = append(out, idt...)
    }
    ...
  }
}
```

Reply:
Sorry for the spelling mistake, in my mean it should be "first". But now your suggest is more slightly cleaner. And I will create PR to replace this.